Server/shared/packets.h

```diff
@@ -146,6 +146,7 @@
 #define WIZ_PACKET17			0x8F
 #define WIZ_DEATH_LIST			0x90
 #define WIZ_CLANPOINTS_BATTLE	0x91 // not sure
+#define WIZ_FOG_DENSITY			0x92 // Controls the Fog density(Sorry for the random place. At some point i believe we'll need to re-organize the packets).
```

Comment:
Not sure how I feel about custom packets. These overlap with official opcodes, so really they should be moved to some unused part of the range (note: the end is taken by special things). I think ~0xca to 0xfd is free.

Reply:
Changed packet to 0xca. Thanks for the heads up so far.